docs/general/userscripts.md

```diff
@@ -13,19 +13,37 @@ AdGuard can significantly broaden website functionality by acting as a userscrip
 
-- **AdGuard Assistant** (legacy version). This custom extension is designed to control filtering directly from the browser page (manual blocking, exclusion of the site, etc.).
+![AdGuard Extra](https://cdn.adtidy.org/content/kb/ad_blocker/general/adguard-extra.png)
+
+#### AdGuard Popup Blocker
+
+The name speaks for itself: it blocks pop-ups — one of the most annoying types of ads on web pages. Learn more about this userscript, its key features, and how to install it on [GitHub](https://github.com/AdguardTeam/PopupBlocker).
+
+![AdGuard Popup Blocker](https://cdn.adtidy.org/content/kb/ad_blocker/general/popup-blocker-installation.png)
+
+#### AdGuard Assistant (legacy version)
+
+This custom extension is designed to control filtering directly from the browser page (manual blocking, exclusion of the site, etc.).
 
 :::note
 
-This version of Assistant is legacy and there is no point in using it on new systems, since it has been replaced with the full-fledged [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). But legacy Assistant might be of help if there is no Browser Assistant extension for your browser.
+This version of Assistant is legacy and there is no point in using it on new systems, since it has been replaced with the full-fledged [Browser Assistant](https://adguard.com/adguard-assistant/overview.html). But the legacy Assistant might be of help if there is no Browser Assistant for your browser. If this is your case, your can learn how to install AdGuard Assistant on [GitHub](https://github.com/AdguardTeam/AdguardAssistant).
 
 :::
 
-- **Disable AMP**. A script, pre-installed only in AdGuard for Android, that disables AMP (Accelerated Mobile Pages) on the Google search results page.
+#### Disable AMP
+
+A script that is only pre-installed in AdGuard for Android. It disables AMP (Accelerated Mobile Pages) on the Google search results page. Learn more about this userscript and how to install it on [GitHub](https://github.com/AdguardTeam/DisableAMP).
+
+![Disable AMP](https://cdn.adtidy.org/content/kb/ad_blocker/general/disable-amp-installation.png)
+
+#### microShield
+
+A regional userscript for people who visit Korean websites. The microShield userscript blocks Ad-Shield advertisement and anti-adblock. Its original source code is available at [https://github.com/seia-soto/userscripts](https://github.com/seia-soto/userscripts) and [AdShield Defuser](https://github.com/seia-soto/adshield-defuser). This userscript can be installed in AdGuard CoreLibs-based appd, Violentmonkey, Tampermonkey, and [quoid/userscripts](https://github.com/quoid/userscripts). Learn more about microShield and how to install it on [GitHub](https://github.com/List-KR/microShield).
```

Comment:
I don't think that the userscript is required for only people who visit Korean websites.
Some international websites started adding Ad-Shield recovery service into their websites.

So, I suggest some changes.

```suggestion
#### microShield

A userscript for people who visit Korean websites and some international websites. The microShield userscript blocks Ad-Shield advertisement and anti-adblock. Its original source code is available at [https://github.com/seia-soto/userscripts](https://github.com/seia-soto/userscripts) and [AdShield Defuser](https://github.com/seia-soto/adshield-defuser). This userscript can be installed in AdGuard CoreLibs-based appd, Violentmonkey, Tampermonkey, and [quoid/userscripts](https://github.com/quoid/userscripts). Learn more about microShield and how to install it on [GitHub](https://github.com/List-KR/microShield).
```

Reply:
If a document will be translated in Korean language, I suggest [this URL](https://community.list-kr.com/docs/AdGuard/Userscripts/microShield) for installation guide.